Code/GraphMol/SmilesParse/SmilesWrite.h

```diff
@@ -301,5 +306,15 @@ inline std::string MolFragmentToCXSmiles(
                                bondSymbols);
 }
 
+// ! \brief returns canonical RWMol including rationalization of stereo groups
+/*!
+  \param mol : the molecule in question.
+
+
+ */
+
+RDKIT_SMILESPARSE_EXPORT std::unique_ptr<RDKit::RWMol> canonicalizeStereoGroups(
```

Comment:
I'm curious why you decided to have this return a new object. It seems like it would be more efficient to modify the molecule in place. If people don't want to change the molecule they have, they can make a copy and then call this function.

Reply:
that makes sense.  I changed it.